tests/unit/services/cookies-test.js

```diff
@@ -267,39 +280,39 @@ describeModule('service:cookies', 'CookiesService', {}, function() {
       });
 
       it('returns undefined for a cookie that was written for another path', function() {
-        this.fakeFastBoot.request.path = '/path';
+        this.subject().set('_fastBoot.request.path', '/path');
```

Comment:
Yep, it actually does not make any difference. In my process to learn how to replicate the problem, I had changed that as doing some experiments. But you are right, it's not needed. Updated.